scalding-core/src/main/scala/com/twitter/scalding/typed/cascading_backend/AsyncFlowDefRunner.scala

```diff
@@ -222,9 +223,24 @@ class AsyncFlowDefRunner extends Writer { self =>
 
     val done = Promise[Unit]()
 
+    val writtenPipes = writes.map {
+      case Force(pipe) => pipe
+      case ToIterable(pipe) => pipe
+      case SimpleWrite(pipe, _) => pipe
+    }
+
+    val depGraph = new DependantGraph[TypedPipe[Any]] {
```

Comment:
it's not. This one has a number of written pipes, the CascadingBackend works for a single pipe.